rest-api-spec/src/main/resources/rest-api-spec/api/open_search_context.json

```diff
@@ -0,0 +1,61 @@
+{
+  "open_search_context":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/search-context.html",
+      "description":"Open a search context that can be used in subsequent searches"
+    },
+    "stability":"beta",
+    "url":{
+      "paths":[
+        {
+          "path":"/_search_context",
```

Comment:
When I am trying to run `/_search_context` or `*/_search_context` or `_all/_search_contex`, I am getting this error
```
{
    "error": {
        "root_cause": [
            {
                "type": "security_exception",
                "reason": "action [indices:data/read/open_search_context] is unauthorized for user [elastic]"
            }
        ],
        "type": "security_exception",
        "reason": "action [indices:data/read/open_search_context] is unauthorized for user [elastic]",
        "caused_by": {
            "type": "illegal_state_exception",
            "reason": "There are no external requests known to support wildcards that don't support replacing their indices"
        }
    },
    "status": 403
}
```

I think it happens because `OpenSearchContextRequest` similar to `SearchRequest` should implement `implements IndicesRequest.Replaceable`, which it doesn't do right now.

Reply:
Great catch. Thanks Mayya. I've added the test and implemented this in https://github.com/elastic/elasticsearch/pull/56480/commits/52902017590b79a3325c66eef3b9e1b47467a9b9.